doc/node.1

```diff
@@ -100,6 +100,10 @@ The default is
 File name of the V8 CPU profile generated with
 .Fl -cpu-prof
 .
+.It Fl -diagnostic-dir
+Set the directory for all diagnostic output files.
```

Comment:
Sync phrasing with markdown suggestion above, and review the docs for the affected options in this man page file. Its a bit more terse, maybe there isn't anything that has to be said here.